cmd/ndm_daemonset/probe/eventhandler.go

```diff
@@ -33,6 +33,8 @@ const (
 	AttachEA EventAction = libudevwrapper.UDEV_ACTION_ADD
 	// DetachEA is detach disk event name
 	DetachEA EventAction = libudevwrapper.UDEV_ACTION_REMOVE
+	// MountEA is mount-point/fs change event
+	MountEA EventAction = "mountch"
```

Comment:
```suggestion
	MountEA EventAction = "mount-change"
```
Better to have meaningful names so we dont confuse it later.